internal/controller/streams.go

```diff
@@ -426,24 +432,57 @@ func (sm *streamManager) StreamConfigurationUpdates(ctx context.Context, logger
 }
 
 // startFlowCacheOutReader starts a goroutine that reads flows from the flow cache and sends them to the cloud secure.
-func (sm *streamManager) startFlowCacheOutReader(ctx context.Context, logger *zap.Logger) error {
+func (sm *streamManager) startFlowCacheOutReader(ctx context.Context, logger *zap.Logger, networkFlowStreamOpen chan struct{}) error {
 	flowCount := 0
-	ticker := time.NewTicker(60 * time.Second) // Create a ticker that triggers every 60 seconds
-	defer ticker.Stop()                        // Ensure the ticker is stopped when the function exits
+	allowedFlows := 200                           // Initialize allowed flows to some default value IE 200, This value is set by the server in the ServerWindow message
```

Comment:
1. I'd pick a more descriptive name, consistent with the new message.
2. Shouldn't this be initialized to zero?! Only the server should be setting the window size. Like in TCP.
```suggestion
	serverWindow := 0 // Number of flow messages that can be sent without blocking until the server increases this window with a ServerWindow message.
```